### DISCUSSION TITLE
Is there any plan to support bls12-381 aggregation  signature verification

### DISCUSSION THREAD

**Discussion by zhengjunhe** [Consensys/gnark#877] [Ideas]
Hi,

As bls12-381 aggregation  signature is used in ethereum consenus layer, I wonder that is there any plan to support it in gnark?

Thanks.

**Comment by ivokub** [comment]
BLS signature verification is usually a single pairing check + hash-to-field. We already have the pairing check for BLS12-381, so it should be possible to implement straightforwardly assuming that the hashed value is verified.

**Reply by zhengjunhe** [reply]
Yes,I see it.

Thanks
